sleap/gui/app.py

```diff
@@ -1339,8 +1339,6 @@ def updateStatusMessage(self, message: Optional[str] = None):
                 hide_key = self.shortcuts["show instances"].toString()
                 message += f" [Hidden] Press '{hide_key}' to toggle."
                 self.statusBar().setStyleSheet("color: red")
-            else:
-                self.statusBar().setStyleSheet("color: black")
 
```

Comment:
Hit `H` to hide instances (the status bar will have red text) and then hit `H` again to show instances (the status bar will still have red text).